cockroachcloud/cloud-sso.md

```diff
@@ -73,8 +73,8 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
 
 ### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
```

Comment:
Please address this linter warning too.

Reply:
It does not apply here. The name of the feature is "Enterprise Authentication", that does not change if we change the name "Enterprise", which is orthogonal to Enterprise Authentication. Is there a way to one off dismiss or over-ride the linter warning?

Reply:
The variable resolves to the word Enterprise so I've been using it in these cases. Another option is to create a Liquid variable for the name of this feature and to use that everywhere. Maybe Nick has more ideas.